### PR TITLE
Remove black exclusion for awx_collection

### PR DESCRIPTION
**Pull request by shanemcd**
We added an exclusion in https://github.com/ansible/awx/commit/c9e7747f2d05aaf0c3cd66303a6f40961f57c42e and I'm not sure why.

### CODE REVIEW

**Comment by AlanCoding** [comment]
This looks unlikely to pass `ansible-test sanity`, which is, unfortunately, not ran as a part of this CI.

**Comment by shanemcd** [comment]
@AlanCoding  Hmmm. They don't seem to pass on devel either. You know if I'm doing something wrong?

```
$ ansible-test sanity --target-python=3.9
WARNING: Ignoring tests in "test/" in favor of "tests/".
Running sanity test "action-plugin-docs"
Running sanity test "ansible-doc"
Running sanity test "changelog"
Running sanity test "compile" on Python 3.9
Running sanity test "empty-init"
Running sanity test "future-import-boilerplate"
Running sanity test "ignores"
Running sanity test "import" on Python 3.9
ERROR: Found 1 import issue(s) on python 3.9 which need to be resolved:
ERROR: plugins/lookup/schedule_rrule.py:103:0: traceback: NameError: name 'rrule' is not defined
See documentation for help: https://docs.ansible.com/ansible-core/devel/dev_guide/testing/sanity/import.html
Running sanity test "line-endings"
Running sanity test "metaclass-boilerplate"
Running sanity test "no-assert"
Running sanity test "no-basestring"
Running sanity test "no-dict-iteritems"
Running sanity test "no-dict-iterkeys"
Running sanity test "no-dict-itervalues"
Running sanity test "no-get-exception"
Running sanity test "no-illegal-filenames"
Running sanity test "no-main-display"
Running sanity test "no-smart-quotes"
Running sanity test "no-unicode-literals"
Running sanity test "pep8"
ERROR: Found 2 pep8 issue(s) which need to be resolved:
ERROR: plugins/modules/organization.py:195:115: E231: missing whitespace after ','
ERROR: plugins/modules/tower_organization.py:195:115: E231: missing whitespace after ','
See documentation for help: https://docs.ansible.com/ansible-core/devel/dev_guide/testing/sanity/pep8.html
Running sanity test "pslint"
Running sanity test "pylint"
Running sanity test "replace-urlopen"
Running sanity test "runtime-metadata"
Running sanity test "shebang"
Running sanity test "shellcheck"
Running sanity test "symlinks"
Running sanity test "use-argspec-type-path"
Running sanity test "use-compat-six"
Running sanity test "validate-modules"
ERROR: Found 39 validate-modules issue(s) which need to be resolved:
ERROR: plugins/modules/tower_ad_hoc_command.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'ad_hoc_command'
ERROR: plugins/modules/tower_ad_hoc_command_cancel.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'ad_hoc_command_cancel'
ERROR: plugins/modules/tower_ad_hoc_command_wait.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'ad_hoc_command_wait'
ERROR: plugins/modules/tower_application.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'application'
ERROR: plugins/modules/tower_controller_meta.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'controller_meta'
ERROR: plugins/modules/tower_credential.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'credential'
ERROR: plugins/modules/tower_credential_input_source.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'credential_input_source'
ERROR: plugins/modules/tower_credential_type.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'credential_type'
ERROR: plugins/modules/tower_execution_environment.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'execution_environment'
ERROR: plugins/modules/tower_export.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'export'
ERROR: plugins/modules/tower_group.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'group'
ERROR: plugins/modules/tower_host.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'host'
ERROR: plugins/modules/tower_import.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'import'
ERROR: plugins/modules/tower_instance_group.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'instance_group'
ERROR: plugins/modules/tower_inventory.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'inventory'
ERROR: plugins/modules/tower_inventory_source.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'inventory_source'
ERROR: plugins/modules/tower_inventory_source_update.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'inventory_source_update'
ERROR: plugins/modules/tower_job_cancel.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'job_cancel'
ERROR: plugins/modules/tower_job_launch.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'job_launch'
ERROR: plugins/modules/tower_job_list.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'job_list'
ERROR: plugins/modules/tower_job_template.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'job_template'
ERROR: plugins/modules/tower_job_wait.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'job_wait'
ERROR: plugins/modules/tower_label.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'label'
ERROR: plugins/modules/tower_license.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'license'
ERROR: plugins/modules/tower_notification_template.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'notification_template'
ERROR: plugins/modules/tower_organization.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'organization'
ERROR: plugins/modules/tower_project.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'project'
ERROR: plugins/modules/tower_project_update.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'project_update'
ERROR: plugins/modules/tower_role.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'role'
ERROR: plugins/modules/tower_schedule.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'schedule'
ERROR: plugins/modules/tower_settings.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'settings'
ERROR: plugins/modules/tower_team.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'team'
ERROR: plugins/modules/tower_token.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'token'
ERROR: plugins/modules/tower_user.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'user'
ERROR: plugins/modules/tower_workflow_approval.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'workflow_approval'
ERROR: plugins/modules/tower_workflow_job_template.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'workflow_job_template'
ERROR: plugins/modules/tower_workflow_job_template_node.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'workflow_job_template_node'
ERROR: plugins/modules/tower_workflow_launch.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'workflow_launch'
ERROR: plugins/modules/tower_workflow_node_wait.py:0:0: invalid-documentation: DOCUMENTATION.module: not a valid value for dictionary value @ data['module']. Got 'workflow_node_wait'
See documentation for help: https://docs.ansible.com/ansible-core/devel/dev_guide/testing/sanity/validate-modules.html
Running sanity test "yamllint"
ERROR: The 3 sanity test(s) listed below (out of 32) failed. See error output above for details.
import --python 3.9
pep8
validate-modules
```

**Comment by AlanCoding** [comment]
ansible-test is very opinionated about the python environment in which it is run. I have a great many issues running locally myself.

**Comment by AlanCoding** [comment]
started off an automated run, should send to you

File: awx_collection/plugins/module_utils/controller_api.py [hunk 1]
```diff
@@ -437,7 +414,7 @@ def fail_wanted_one(self, response, endpoint, query_params):
         if len(sample['json']['results']) > 1:
             sample['json']['results'] = sample['json']['results'][:2] + ['...more results snipped...']
         url = self.build_url(endpoint, query_params)
-        display_endpoint = url.geturl()[len(self.host):]  # truncate to not include the base URL
+        display_endpoint = url.geturl()[len(self.host) :]  # truncate to not include the base URL
```
Comment (AlanCoding) [inline awx_collection/plugins/module_utils/controller_api.py:417]: looks like this line fails pep8

**Pull request closed without merging**